src/components/GlobeOptimized.jsx

```diff
@@ -395,20 +398,44 @@ export default function GlobeOptimized(props) {
       properties.forEach((property) => {
         switch (property) {
           case 'lat':
+            console.log('Rotation -->');
+            console.log(rotation);
             rotation[1] +=
               params.has(property) && !isNaN(params.get(property))
                 ? Number(params.get(property))
                 : 0;
+            //if initial longitude and latitude not in the range rest it
+            if(rotation[1] <-360 || rotation[1] > 360){
+              rotation[1] = 0;
+              rotation[0] = 180;
```

Comment:
Also why are we change `rotation[0]` when we verify the value for `rotation[1]` ?

Reply:
If any of these values are out of the bound we are setting the globe to its initial position.